### PR TITLE
Fix TestCommitLog.ignore_failure_policy_test on Windows

### PR DESCRIPTION
**Pull request by josh-mckenzie**
os.chmod can't be trusted on Windows. Comments on the PR should be pretty explanatory.

### CODE REVIEW

File: commitlog_test.py [hunk 1]
```diff
@@ -245,12 +246,22 @@ def ignore_failure_policy_test(self):
...
-              INSERT INTO test (key, col1) VALUES (2, 2);
-            """)
-        # Should not exists
-        assert_none(self.session1, "SELECT * FROM test where key=2;")
+        # on Windows, we can't delete the segments if they're chmod to 0 so they'll still be available for use by CLSM,
+        # and we can still create new segments since os.chmod is limited to stat.S_IWRITE and stat.S_IREAD to set files
+        # as read-only. New mutations will still be allocated and WriteTimeouts will not be raised. It's sufficient that
+        # we confirm that a) the node isn't dead (stop) and b) the node doesn't terminate the thread (stop_commit)
+        query = "INSERT INTO test (key, col1) VALUES (2, 2);"
+        if platform.system() == 'Windows':
```
Comment (ptnapoleon) [inline commitlog_test.py]: :'(. Why not import is_win from ccmlib.common and call that?

**Comment by josh-mckenzie** [comment]
... I have no reason not to actually.

I'm going to take a wack at fixing the stop_* policy test as well so I'll switch to the is_win check on that and update this PR.

**Comment by josh-mckenzie** [comment]
Added a second commit that imports is_win and also fixes the stop_failure test. Just needed a touch more data in the stress population to make sure the DB exhausted its current CL segment on Windows for some reason, so doubled the record count on populate (was low to begin with)

File: commitlog_test.py [hunk 2]
```diff
@@ -128,10 +129,10 @@ def _provoke_commitlog_failure(self):
 
         if self.cluster.version() < "2.1":
             with open(os.devnull, 'w') as devnull:
-                self.node1.stress(['--num-keys=500000'], stdout=devnull, stderr=subprocess.STDOUT)
+                self.node1.stress(['--num-keys=1000000'], stdout=devnull, stderr=subprocess.STDOUT)
         else:
             with open(os.devnull, 'w') as devnull:
-                self.node1.stress(['write', 'n=500000', '-rate', 'threads=25'], stdout=devnull, stderr=subprocess.STDOUT)
+                self.node1.stress(['write', 'n=1000000', '-rate', 'threads=25'], stdout=devnull, stderr=subprocess.STDOUT)
```
Comment (ptnapoleon) [inline commitlog_test.py]: Can you make this `n=1M` for readability please?

**Comment by josh-mckenzie** [comment]
Done. I didn't realize it accepted that nomenclature - I'll be using that in the future.

**Comment by ptnapoleon** [comment]
+1

**Pull request merged**